experimental/README.md

```diff
@@ -19,7 +19,7 @@ To enable experimental features, start the Docker daemon with the
 
 ```json
 {
-    "experimental": true
+    "experimental": "enabled"
```

Comment:
Hm, this is for the _daemon_ configuration, not the CLI. The Cli uses "enabled" / "disabled", but the daemon uses a boolean

Reply:
Ah ha, got it, thanks. That's horrible that they're different, but understood. I'll close this off.